CMakeLists.txt

```diff
@@ -108,6 +108,18 @@ option(BUILD_EXAMPLE "Build examples" OFF)
 option(ONLY_INSTALL "Only install" OFF)
 option(BUILD_CODE_COVERAGE "Build with code coverage enabled" OFF)
 
+# FOR HANDLING ENABLE/DISABLE OPTIONAL BACKWARD COMPATIBILITY for FILE/FOLDER REORG
+option(BUILD_FILE_REORG_BACKWARD_COMPATIBILITY "Build with file/folder reorg with backward compatibility enabled" OFF)
+if(BUILD_FILE_REORG_BACKWARD_COMPATIBILITY)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DFILE_REORG_BACKWARD_COMPATIBILITY")
+  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -DFILE_REORG_BACKWARD_COMPATIBILITY")
```

Comment:
These CMAKE_CXX_FLAGS and CMAKE_C_FLAGS may trigger rocm-checks warnings and why do you need them as I don't see you using those defines within the actual c/c++ files?

Reply:
Agreed with @TorreZuk.  I also don't see FILE_REORG_BACKWARD_COMPATIBILITY being used anywhere within the actual library code.

Reply:
Removed these unused setting in the latest